src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/VedtaksperioderMedBegrunnelser/BegrunnelserMultiselect.tsx

```diff
@@ -27,17 +26,16 @@ import { useVedtaksperiodeMedBegrunnelser } from '../Context/VedtaksperiodeMedBe
 import { mapBegrunnelserTilSelectOptions } from '../Hooks/useVedtaksbegrunnelser';
 
 interface IProps {
-    vedtaksperiodetype: Vedtaksperiodetype;
+    ikkeRedigerbar: boolean;
 }
 
 const GroupLabel = styled.div`
     color: black;
 `;
 
-const BegrunnelserMultiselect: React.FC<IProps> = ({ vedtaksperiodetype }) => {
+const BegrunnelserMultiselect: React.FC<IProps> = ({ ikkeRedigerbar }) => {
     const { vurderErLesevisning } = useBehandling();
-    const skalIkkeEditeres =
-        vurderErLesevisning() || vedtaksperiodetype === Vedtaksperiodetype.AVSLAG;
+    const skalIkkeEditeres = vurderErLesevisning() || ikkeRedigerbar;
```

Comment:
Vi burde egentlig ungå negative navn for boolske variabler.
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Reply:
Helt enig. Endret begge variablene du kommenterte på: 462efd8